components/Main.jsx

```diff
@@ -19,17 +20,23 @@ const Main = () => {
           </h1>
           <div className="flex items-center justify-between max-w-[330px] m-auto py-4">
             <div className="rounded-full shadow-lg shadow-gray-400 p-6 cursor-pointer hover:scale-110 ease-in duration-300">
-              <FaLinkedinIn />
+              <a
+                href="https://www.linkedin.com/in/audreyfocant/"
+                target="_blank"
+              >
+                <FaLinkedinIn />
+              </a>
             </div>
             <div className="rounded-full shadow-lg shadow-gray-400 p-6 cursor-pointer hover:scale-110 ease-in duration-300">
-              <FaGithub />
+            <a href="https://github.com/Faudrey28 " target="_blank">
+                      <FaGithub />
```

Comment:
This tag doesn't look aligned the same way others are on the page. Could you beautify the code one more time?